src/app/containers/ArticleTimestamp/index.jsx

```diff
@@ -52,7 +57,7 @@ const ArticleTimestamp = ({ firstPublished, lastPublished, popOut }) => {
   const Wrapper = popOut ? PopOutGridItemMedium : GridItemConstrainedMedium;
 
   return (
-    <Wrapper>
+    <Wrapper className={className}>
```

Comment:
Would it be possible to add a snapshot test for the article timestamp - I checked the code quickly, and there doesn't appear to be one.  It would be nice to have one as a fallback.

Reply:
Added two snapshot tests - one for when there's only a 'created' timestamp, another for when there's both a 'created' and 'updated' timestamps.